src/app.tsx

```diff
@@ -33,11 +36,8 @@ export default function App() {
   );
 
   return (
-    <Provider store={store}>
-      <ThemeProvider>
-        <Router />
-        {githubButton}
-      </ThemeProvider>
-    </Provider>
```

Comment:
don' t remove provider.
this is to provide redux state access to the application.